packages/core/src/TagsInput/TagsInput.js

```diff
@@ -200,6 +198,7 @@ const HvTagsInput = (props) => {
                 ...value.slice(tagCursorPos + 1),
               ];
               setValue(newTagsArr);
+              setResetInput(false);
```

Comment:
I understand: the reset shouldn't occur when deleting. But why would it be true? Maybe because it wasn't reset back to false in the useEffect?